packages/contracts/CHANGELOG.md

```diff
@@ -5,7 +5,7 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## v1.4.0-rc0
+## v1.4.0
```

Comment:
this shouldn't be [upcoming] ?

Reply:
Not necessarily. I think it is clearer for us in which release things will end up and if the version number is already written down. However, I am open to change it